src/lib/markbind/src/parsers/componentParser.js

```diff
@@ -119,24 +139,61 @@ function _assignPanelId(element) {
   }
 }
 
+/*
+ * Triggers
+ */
+
+function _parseTrigger(element) {
+  const el = element;
```

Comment:
Same for `el` and `element`

Reply:
Also, in the above we seem to be favouring explicitness (`child` instead of `c`), but here we seem to be favouring terseness. Is there a reason for this change?

Reply:
I'll change the child function then, to keep it more consistent.